ludwig/data/preprocessing.py

```diff
@@ -1661,6 +1669,62 @@ def _handle_missing_values(
         raise ValueError(f"Invalid missing value strategy {missing_value_strategy}")
 
 
+def handle_llm_prompt_injection(
+    dataset_cols, feature_configs, feature_name_to_preprocessing_parameters, backend
+) -> None:
+    """If output feature is a category feature and it's preprocessing has prompt_template, then we need to inject
+    the prompt into the input text feature(s)."""
```

Comment:
Can you put an example of the input prompt and output prompt in the comment?

Reply:
Added!